instrumentation/elastic-opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai/__init__.py

```diff
@@ -95,6 +97,10 @@ def _instrument(self, **kwargs):
         )
 
     def _uninstrument(self, **kwargs):
+        # unwrap only supports uninstrementing real module references so we
+        # import here.
+        import openai
```

Comment:
This is unusual (directly importing the thing in an `_uninstrument`), right?

Reply:
Yeah, since openai depends on httpx if we load the openai module earlier and we are loaded before the httpx instrumentation the http spans from the openai client won't be traced